.github/workflows/codeql.yml

```diff
@@ -3,9 +3,6 @@ name: "CodeQL"
 on:
   push:
     branches: [ "master" ]
-  schedule:
-    - cron: '0 3 * * *'
-      displayName: Daily night check
```

Comment:
The pipeline doesn't want to start at 03:00, so change it to start simply after each new commit in master